security-common/src/main/kotlin/org/octopusden/cloud/commons/security/client/AuthServerClient.kt

```
@@ -45,18 +45,18 @@ class AuthServerClient(
                 ?.let { openIdConfigurationUrl ->
-                    restTemplate.getForEntity(openIdConfigurationUrl, OpenIdConfiguration::class.java)
+                    restTemplate
+                        .getForEntity(openIdConfigurationUrl, OpenIdConfiguration::class.java)
                         .body ?: throw OAuth2AuthenticationException(
                         OAuth2Error("invalid_request"),
-                        "Cannot Extract openid-configuration via $openIdConfigurationUrl"
+                        "Cannot Extract openid-configuration via $openIdConfigurationUrl",
                     )
                 } ?: throw BeanInitializationException("Open ID Configuration URL must be provided")
-
         } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is too generic. Prefer catching specific exceptions to the case that is currently handled.

(detekt.exceptions.TooGenericExceptionCaught)